bin/fetch-historic-measurements.js

```diff
@@ -0,0 +1,289 @@
+/*
+Usage:
+
+1. Setup port forwarding between your local computer and Postgres instance hosted by Fly.io
+  ([docs](https://fly.io/docs/postgres/connecting/connecting-with-flyctl/)). Remember to use a
+  different port if you have a local Postgres server for development!
+   ```sh
+   fly proxy 5454:5432 -a spark-db
+   ```
+
+2. Find spark-db entry in 1Password and get the user and password from the connection string.
+
+3. Run the following command to fetch all measurements, remember to replace "user" and "password"
+   with the real credentials:
+
+   ```sh
+   DATABASE_URL="postgres://user:password@localhost:5454/spark" node bin/fetch-historic-measurements.js <range-start> <range-end>
+   ```
+
+   This will fetch all measurements committed between range-start (inclusive) and range-end (exclusive)
+   and write them to file.
+*/
+
+// dotenv must be imported before importing anything else
+import 'dotenv/config'
+
+import { Point } from '@influxdata/influxdb-client'
+import * as Sentry from '@sentry/node'
+import createDebug from 'debug'
+import fs from 'node:fs'
+import { mkdir, readFile, writeFile } from 'node:fs/promises'
+import os from 'node:os'
+import path from 'node:path'
+import pMap from 'p-map'
+import { fetchMeasurements, preprocess } from '../lib/preprocess.js'
+import { RoundData } from '../lib/round.js'
+import * as SparkImpactEvaluator from '@filecoin-station/spark-impact-evaluator'
+import pg from 'pg'
+
+const { DATABASE_URL } = process.env
+
+Sentry.init({
```

Comment:
Why does this script need to report errors to Sentry? Do you envision we will be running it automatically in an unobserved way? If not, can we remove Sentry integration?